Source/Plugin.BLE/Android/Device.cs

```diff
@@ -427,5 +427,68 @@ protected override bool UpdateConnectionIntervalNative(ConnectionInterval interv
 #endif
         }
 
+
+        protected override DeviceBondState GetBondState()
+        {
+            if (NativeDevice == null)
+            {
+                Trace.Message($"[Warning]: Can't get bond state of {Name}. NativeDevice is null.");
+                return DeviceBondState.NotSupported;
+            }
+            var bondState = NativeDevice.BondState;
+            switch (bondState)
+            {
+                case global::Android.Bluetooth.Bond.None:
+                    return DeviceBondState.NotBonded;
+                case global::Android.Bluetooth.Bond.Bonding:
+                    return DeviceBondState.Bonding;
+                case global::Android.Bluetooth.Bond.Bonded:
```

Comment:
Here are three more cases of an unnecessary prefix `global::Android.Bluetooth`.

Also: Maybe the conversion from the native states could be encapsulated into a (static) method `DeviceBondState.FromNative`? (Similar to the `ToNative` extensions we already have, e.g. for the enums `ScanMode` and `ScanMatchMode`.)
Or rather an extension method `ToDeviceBondState` for the enum `Android.Bluetooth.Bond`? (Analogous to the method `ToBluetoothState`.)

This would make the code even clearer and would reduce code duplication with `BroadcastReceiver.OnReceive`.

Reply:
As your commit 92874a4d0489be3ee0a8560e5c10f3e13aabd249 shows, this indeed makes the code much cleaner. Pretty nice! 👍 

You could even save the extra local variable `bondState` in `OnReceive` now (but ok, I'm getting really picky now; please stop me before it gets too bad 😆)

Reply:
The compiler should remove this temporary variable in release builds.
I prefer to keep it for readability.
In general I prefer that a maintainer looks into PRs so don't worry. I also like to respect to coding style of a project. But there is not always enough time.

Reply:
> The compiler should remove this temporary variable in release builds. I prefer to keep it for readability.

Fully ok with me 😊 
(Such things are pretty much a matter of taste, and as a reviewer I'm merely trying to make suggestions, but not enforce my personal taste everywhere.)